data_steward/analytics/cdr_ops/ehr_snapshot_qc.py

```diff
@@ -18,12 +19,16 @@
 PROJECT_ID = ""  # identifies the project containing the datasets
 BASELINE_EHR_DATASET_ID = ""  # Identifies the dataset the snapshot was created from
 EHR_SNAPSHOT_DATASET_ID = ""  # Identifies the snapshot dataset
+RELEASE_TAG = ""  # identifies the release tag for the current CDR release
+EXCLUDED_SITES = "default"  # List of excluded sites passed as string: eg. "'hpo_id1', 'hpo_id_2', 'hpo_id3',..."
```

Comment:
This feels a bit odd. This may be better as a list of hpo_ids than a string.

Reply:
Papermill takes only strings as input for parameters, that's the reason I have to use this notation. I tried to pass a list but it didn't work.

Reply:
Ah I see, thanks! I see a hardcoded release tag, other than that the ticket looks good.